CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 7.10.0 (Swift PM)
```

Comment:
```suggestion
# 7.10.0
```

Reply:
Thanks for confirming. I was wondering that.

Reply:
yep, for sure!